### DISCUSSION TITLE
reroute every jamulus channel to external mixer (x32/wing/reaper) and reinsert the master into jamulus or zoom

### DISCUSSION THREAD

**Discussion by Rabski** [jamulussoftware/jamulus#2212] [Technical Support & Help]
Is it possible to send every jamulus track to an external mixer (x32/Wing/SQ) so that the soundengineer can adjust audio per channel.

The new combined audio can than be routed back to Jamulus or Zoom. 

Greetings,

Willem

**Comment by pljones** [comment]
The mixer is at the server, not the client.  The client has two audio in channels and two audio out channels -- from the server.  The client also allows all of these to be connected to a supported external audio system.

The client mixer supports MIDI automation, though, so it's possible to use a hardware _controller_ to control the faders and pan settings (of a limited set of channels).  (Note that this feature is not supported on Windows.)

**Comment by chrisrimple** [comment]
The [WorldJam](http://worldjam.vip) accomplishes this by doing the mixing in a Jamulus client, then outputting to a web stream.
